xbmc/games/addons/GameClientProperties.h

```diff
@@ -77,7 +77,7 @@ class CGameClientProperties
   bool HasProxyDll(const std::string& strLibPath) const;
 
   const CGameClient* const  m_parent;
-  game_client_properties    m_properties;
+  AddonProps_Game&          m_properties;
```

Comment:
This var is a hack from early in game development, entirely to avoid modifying CAddonDll to keep the size of the patch down. I'd be glad to see it go, so +1 from me.